webapp/i18n/en.json

```diff
@@ -1836,6 +1836,7 @@
   "user.settings.display.showNickname": "Show nickname if one exists, otherwise show first and last name",
   "user.settings.display.showUsername": "Show username (default)",
   "user.settings.display.teammateDisplay": "Teammate Name Display",
+  "user.settings.display.theme.allMyThemes": "Apply new theme to all my teams",
```

Comment:
Thanks for catching that the message didn't exist in the localization files, but the key is mismatched. It should be `user.settings.display.theme.applyToAllTeams`